src/main/java/uk/gov/hmcts/reform/sscs/helper/mapping/HearingsPartiesMapping.java

```diff
@@ -164,13 +167,16 @@ public static Optional<String> getIndividualPreferredHearingChannel(String heari
     }
 
     public static Optional<String> getIndividualInterpreterLanguage(HearingOptions hearingOptions) {
-        if (isTrue(hearingOptions.wantsSignLanguageInterpreter())) {
-            return getSignLanguage(hearingOptions)
-                .map(SignLanguage::getHmcReference);
-        }
-        if (isYes(hearingOptions.getLanguageInterpreter())) {
-            return getInterpreterLanguage(hearingOptions)
-                .map(InterpreterLanguage::getHmcReference);
+
+        if (nonNull(hearingOptions)) {
+            if (isTrue(hearingOptions.wantsSignLanguageInterpreter())) {
+                return getSignLanguage(hearingOptions)
+                    .map(SignLanguage::getHmcReference);
+            }
+            if (isYes(hearingOptions.getLanguageInterpreter())) {
+                return getInterpreterLanguage(hearingOptions)
+                    .map(InterpreterLanguage::getHmcReference);
+            }
         }
         return Optional.empty();
```

Comment:
Best to get out early. It also reduces nested statements.
I.e.
if(isNull(hearingOptions)) {
return Optional.empty();
{